shell/platform/android/io/flutter/plugin/platform/ImageReaderPlatformViewRenderTarget.java

```diff
@@ -53,9 +53,9 @@ public void onImageAvailable(ImageReader reader) {
   protected ImageReader createImageReader33() {
     final ImageReader.Builder builder = new ImageReader.Builder(bufferWidth, bufferHeight);
     // Allow for double buffering.
-    builder.setMaxImages(2);
-    // Assume that we will be producing 32-bit RGBA values.
-    builder.setDefaultHardwareBufferFormat(HardwareBuffer.RGBA_8888);
+    builder.setMaxImages(3);
```

Comment:
Are there any downsides to unconditionally requesting private format here? i.e. does it stop us from capturing the bytes via readback on the framework side (Picture/Layer.toImage)

Reply:
This is a good point, it definitely will stop us from reading it back.

I'm going to leave a TODO here to consider this in the future.